src/pages/ReimbursementAccount/ACHContractStep.js

```diff
@@ -164,7 +196,6 @@ class ACHContractStep extends React.Component {
             BankAccounts.updateReimbursementAccountDraft(newState);
```

Comment:
I don't think `toggleCheckbox` is being used anymore. We can remove it if that's the case.